lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/update-name.yaml

```diff
@@ -180,6 +180,10 @@ states:
       access-denied:
         targetJourney: FAILED
         targetState: FAILED_CONFIRM_DETAILS
+        checkJourneyContext:
+          rfc:
+            targetJourney: FAILED
+            targetState: FAILED_CONFIRM_DETAILS
```

Comment:
I added so that if the `updateDetailsAccountDeletion` flag is enabled it wont drop into the FAILED_UPDATE_DETAILS for rfc journeys as the context takes precedence. But happy to switch the check to the failure if it makes more sense

Reply:
I think so - conceptually there are 3 changes in the fast-follow, even if it's not exactly how the tickets have been defined:
- Certain events from dcmaw (e.g. this `access-denied`) should display a new page, which we've called `FAILED_UPDATE_DETAILS`, while other events (e.g. `fail-with-ci`) still route to `FAILED_CONFIRM_DETAILS`
  - This is what the feature flag check below is doing
- `FAILED_CONFIRM_DETAILS` now has two variants, for reuse vs. repeat fraud check
  - I think this should probably just live in `failed.yaml`
- `FAILED_UPDATE_DETAILS` also has two variants, for reuse vs. repeat fraud check
  - similarly

So this file decides which type of failure it is (confirm vs. update - not great names but never mind..!), and the failures decide which variant of that you see.

Reply:
Done :)